cip-00XX/cip-XXXX.md

```diff
@@ -0,0 +1,120 @@
+## Weighted Validator Liveness Rewards for SV-Determined Parties
+
+<pre>
+  CIP: ?
+* Layer: Daml
+  Title: Weighted Validator Liveness Rewards for SV Chosen Parties
+  Author: Moritz Kiefer
+* Discussions-To: cip-discuss@list.sync.global
+* Comments-Summary: N/A
+* Comments-URI: N/A
+  Status: Draft
+  Type: Tokenomics
+  Created: 2025-07-23
+  License: CC0-1.0
+* Post-History: N/A
+</pre>
+
+## Abstract
+
+This CIP proposes extending the concept of validator liveness rewards
+to arbitrary parties not just validator operators as well as
+supporting weights defined through governance.
+
+This allows multi-hosting of validators while still providing the same
+rewards to the parties on the same node as if they were running their
+own node. It also allows weighting certain some parties higher, e.g.,
+because they have been shown to contribute significantly to the
+success of the network.
+
+## Copyright
+
+This CIP is licensed under CC0-1.0: [Creative Commons CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/)
+
+Note: other licenses are avilable see recommendations in [cip-0000](../../cips/cip-0000/cip-0000.md)
+
+## Specification
+
+The existing functionality where each SV can unilaterally grant a
+`ValidatorLicense` to any validator operator by providing a validator
+onboarding secret, is extended such that each SV can unilaterally
+grant a `ValidatorLicense` with the default weight to arbitrary
+parties (including parties relying on external signing) through the SV
+UI.
+
+In addition to that, SVs get the ability to change the weight of a
+validator license to a non-negative integer. The liveness rewards
+minted by a license with weight N are equivalent to the liveness
+rewards minted by N licenses.
+
+### SV UI
+
+The SV UI is extended with three different functionalities:
+
+1. An SV can grant a validator license to a user-specified party id.
+2. SVs can vote on weight changes for an existing validator license.
+3. SVs can vote on withdrawing a validator license.
+
+### Daml
+
+- At the Daml level there is no distinction between validator operator
+  parties and any other parties. So the existing choices that allow an
+  SV to grant a validator license to validator operator parties can
+  already be used to grant licenses to arbitrary parties.
+- To support parties relying on external signing, a new
+  `MintingDelegation` contract is added which allows signing an
+  transaction once which delegates the right to create and mint
+  liveness activity records to a validator operator party. This then
+  allows the automation in the validator app to submit the required
+  transactions each round on behalf of the external party. Note that
+  `MintingDelegation` is limited to creating and minting activity
+  records, it does not grant the delegate rights to do anything else
+  on behalf of the party.
```

Comment:
How are these `MintingDelegation` contracts accepted and renewed? UI? Config? API call?